### ISSUE TITLE
Integrate DSi camera driver

### ISSUE THREAD

**Issue by AntonioND** [blocksds/sdk#11]
We need to contact the author and relicense it under the Zlib license before we can integrate it:

https://github.com/Epicpkmn11/dsi-camera

**Comment by AntonioND** [comment]
@Epicpkmn11 would you be open to relicense the code?

**Comment by Epicpkmn11** [comment]
Yeah, I'm fine with whatever license, my intention in doing unlicense is to make it free to use however, but if you need me to explicitly say it then yeah zlib's fine

**Comment by AntonioND** [comment]
Perfect, thanks! I'm a bit worried about the Unlicense, it seems to be considered less reliable than something like Creative Commons 0. https://en.wikipedia.org/wiki/Unlicense#Reception

**Comment by Epicpkmn11** [comment]
I suppose I coulda done that on dsi-camera, I usually do CC0 in `LICENSE` files and Unlicense inline just cause it's smaller

I wish it was easier to just declare something public domain, do literally whatever

**Comment by AntonioND** [comment]
Yes, CC0 is the closest you can get. I normally use SPDX tags:

```
// SPDX-License-Identifier: Zlib
//
// Copyright (c) 2023 Antonio Niño Díaz
```

Even the Linux kernel has switched to using this system.

**Comment by AntonioND** [comment]
I'm going to close this task because we have a working driver now: https://github.com/blocksds/libnds/commit/32b071d069f6194e3feee410697eea67000de9dd

I've created a new issue to keep track of improvements: https://github.com/blocksds/sdk/issues/41
